packages/cx-core/src/widgets/form/LookupField.scss

```diff
@@ -73,7 +75,7 @@
    }
 
    .#{$element}#{$name}-tag-value {
-      padding: 0 $cx-default-input-tag-spacing;
+      padding: 0 $cx-default-input-tag-spacing + round($cx-default-tag-border-radius/4);
```

Comment:
Use cx-calc to ensure it works in case if units are different.
`padding: 0 cx-calc($cx-default-input-tag-spacing, round($cx-default-tag-border-radius/4));